### PR TITLE
Authoring: Revert the apearnce and styles of the Editor fields to the way they looked before refactoring

### PR DESCRIPTION
**Pull request by dzonidoo**
SDESK-7249

### CODE REVIEW

File: scripts/apps/authoring/styles/themes.scss [hunk 1]
```diff
@@ -373,6 +373,15 @@ body, html {
         }
     }
 
+    .headline .public-DraftEditor-content {
```
[the remark this answers is not in the document]
Comment (dzonidoo) [reply scripts/apps/authoring/styles/themes.scss:376]: on react

[the remark this answers is not in the document]
Comment (tomaskikutis) [reply scripts/apps/authoring/styles/themes.scss:376]: where does `.headline` class get outputted in react?

[the remark this answers is not in the document]
Comment (dzonidoo) [reply scripts/apps/authoring/styles/themes.scss:376]: i want to sand screen to u but i realised that im wrong... `.headling` is on old authoring.... angular one
but whats your point?

[the remark this answers is not in the document]
Comment (tomaskikutis) [reply scripts/apps/authoring/styles/themes.scss:376]: I'll need to delete it after we switch to authoring-react so I want to keep track of the difference. Also I wanted to check that you're doing things correctly - you shouldn't be using this approach in authoring-react. Speaking of authoring-react - did you test that things look good there?

**Review by tomaskikutis — APPROVED** [review]

**Pull request merged**